packages/conf-assimp/conf-assimp.1/opam

```diff
@@ -0,0 +1,33 @@
+opam-version: "2.0"
+maintainer: "geoffderosenroll@gmail.com"
+authors: ["Geoff deRosenroll"]
+homepage: "https://github.com/assimp/assimp"
+bug-reports: "https://github.com/ocaml/opam-repository/issues"
+license: "BSD-3-Clause"
+build: [
+  ["assimp" "--help"] {os != "openbsd"}
+  ["/usr/local/bin/assimp" "--help"] {os = "openbsd"}
+]
+depexts: [
+  ["assimp-utils"] {os-family = "debian"}
+  ["libassimp-dev"] {os-family = "debian"}
+  ["assimp"] {os-distribution = "arch"}
+  ["assimp-devel"] {os-distribution = "fedora"}
+  ["assimp-devel"] {os-distribution = "ol"}
+  ["assimp"] {os-distribution = "suse"}
+  ["libassimp-dev"] {os-family = "suse"}
+  ["lib64assimp-devel"] {os-distribution = "mageia"}
+  ["assimp"] {os-distribution = "alpine"}
+  ["assimp"] {os-distribution = "nixos"}
+  ["assimp-5.0.0"] {os = "openbsd"}
+  ["assimp"] {os = "macos" & os-distribution = "homebrew"}
+  ["assimp"] {os = "macos" & os-distribution = "macports"}
+  ["assimp"] {os = "freebsd"}
+  ["assimp"] {os = "win32" & os-distribution = "vcpkg"}
```

Comment:
This can be simplified this way. Also `assimp` is the package name for openbsd. There is no package named `assimp-5.0.0`
```suggestion
  ["assimp-utils" "libassimp-dev"] {os-family = "debian"}
  ["assimp-utils" "libassimp-dev"] {os-family = "ubuntu"}
  ["assimp"] {os-family = "arch"}
  ["assimp-devel"] {os-distribution = "fedora"}
  ["assimp-devel"] {os-distribution = "ol"}
  ["assimp" "libassimp-dev"] {os-family = "suse"}
  ["lib64assimp-devel"] {os-distribution = "mageia"}
  ["assimp"] {os-family = "alpine"}
  ["assimp"] {os-distribution = "nixos"}
  ["assimp"] {os = "openbsd"}
```

Reply:
Ah TIL thanks, I should have tried that / read the docs better.